### PR TITLE
{system}[system/system] OpenSSL v3

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Depends on:
- [x] https://github.com/easybuilders/easybuild-easyblocks/pull/3081
- [x] https://github.com/easybuilders/easybuild-easyconfigs/pull/18896

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3081
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-93 - Linux Rocky Linux 9.3 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.18
See https://gist.github.com/SebastianAchilles/b1a0246adb8febcd9bd7db8623169836 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3081
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-89 - Linux Rocky Linux 8.9, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/021bcc4954ab58cd7dc212c0bf6731f5 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3081
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/a5292f2f722a48ca0abb62605e202447 for a full test report.

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# this version of OpenSSL, the installation directory of this module will be
+# populated with symlinks to the system files. The minimum required version of
+# OpenSSL can be finely controled with 'minimum_openssl_version' (defaults to
+# easyconfig version).
+# If the host system does not have this version of OpenSSL (or with the option
+# wrap_system_openssl = False), EasyBuild will fall back to the following
+# component list, which will be build and installed as usual.
+
+components = [
+    (name, '3.2.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.eb:31]: @SebastianAchilles Let's bump this to `3.2.1`?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.eb]: Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19623/commits/56793bccdb8f0ad1b9b575e8e549ccafd7eb329e

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3081
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-93 - Linux Rocky Linux 9.3 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.18
See https://gist.github.com/SebastianAchilles/6ec2d88c7d2deb82ab888eed16f5c53f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3081
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-89 - Linux Rocky Linux 8.9, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/9d3496d7d71bdaff5d2e9ad60ec4f1c4 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsczen3g1.int.jsc-zen3.fz-juelich.de - Linux Rocky Linux 9.3, x86_64, AMD EPYC-Milan Processor (zen3), 1 x NVIDIA NVIDIA A100 80GB PCIe, 545.23.08, Python 3.9.18
See https://gist.github.com/SebastianAchilles/034345fb79e4ab73ee792ef4161daafc for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jscclxc1.int.jsc-clx.fz-juelich.de - Linux Rocky Linux 9.3, x86_64, Intel Xeon Processor (Cascadelake) (cascadelake), Python 3.9.18
See https://gist.github.com/SebastianAchilles/c3714a5ddcc15e3052af07b7b5568805 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 2922"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 2922"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3110.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/779ec359c7835a67b2fef21110271ba6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4230.shinx.os - Linux RHEL 9.2, x86_64, AMD EPYC 9654 96-Core Processor, Python 3.9.16
See https://gist.github.com/boegel/36b7843d1a1c16613ac8444811b7c8f2 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**